resource/css/_page.scss

```diff
@@ -146,18 +150,15 @@
   .main.grant-specified,
   .main.grant-owner {
     background: #333;
-    padding: 0 16px 16px 16px;
```

Comment:
Why removed this padding?

Original:
![image](https://user-images.githubusercontent.com/29064/66360546-84a16480-e9b6-11e9-9fa6-929ec04b3395.png)


Current branch:
![image](https://user-images.githubusercontent.com/29064/66360555-8d923600-e9b6-11e9-806a-ebd2d9b11482.png)


The style looks a bit broken ...

Reply:
In particular, I fixed this because the display width on smartphones become narrower.

Reply:
I've just fixed broken styles. The modified styles looks like this:
![localhost_3000_user_lightnet328_memo_2019_10_12 (1)](https://user-images.githubusercontent.com/2351326/66711760-e570d880-edcc-11e9-9e00-6093f92f73db.png)